### ISSUE TITLE
Load training curve after training finished

### ISSUE THREAD

**Issue by insikk** [rdevon/DIM#33]
While training is going on, I can check the training curve via visdom.
However, I accidentally closed visdom server, and all training logs are gone. 
Some models are still on training, so newly opened visdom server shows their training curve. 
However, already-finished-model's training curve is lost. 

Is there a way to get displayed train-finished-model's training curve on my visdom server?

**Comment by insikk** [comment]
I was able to retrieve evaluation metrics from saved file format. 

```python

import torch
import os

def smooth(scalars, weight):  # Weight between 0 and 1
    last = scalars[0]  # First value in the plot (first timestep)
    smoothed = list()
    for point in scalars:
        smoothed_val = last * weight + (1 - weight) * point  # Calculate smoothed value
        smoothed.append(smoothed_val)                        # Save it
        last = smoothed_val                                  # Anchor the last smoothed value

    return smoothed

plt.plot(test_result['classifier.glob[-1](64,).accuracy_top_1'])        
plt.show()

smoothed_val = smooth(test_result['classifier.glob[-1](64,).accuracy_top_1'], 0.99)
plt.plot(smoothed_val)        
plt.show()

output_path = "./output"

experiments = os.listdir(output_path)
# print(experiments)
for exp_name in experiments:
    exp_path = os.path.join(output_path, exp_name)
    if exp_name[0] == '.':
        continue
    saved_model_path = os.path.join(exp_path, "binaries")
    for saved_model_name in os.listdir(saved_model_path):
        if not saved_model_name.endswith(".t7"):
            continue
        if '_last' in saved_model_name or \
           '_final' in saved_model_name:
            save_path = os.path.join(saved_model_path, saved_model_name)
            graph_path = os.path.join(exp_path, f"{saved_model_name}_global_test_acc.png")
            if os.path.exists(graph_path):
                continue
            print(saved_model_name)
            try:
                model = torch.load(save_path)
            except EOFError as e:                
                print(f"skip processing this model {saved_model_name} due to exception:", e)
                continue
            # print(model.keys())
            test_result = model['results']['results']['results']['test']
            keys = ['classifier.glob[-1](64,).accuracy_top_1',
                    'classifier.glob[-1](128,).accuracy_top_1',
                    'classifier.glob[8](192,5,5).accuracy_top_1',
                   ]
            t_result = None
            for key in keys:
                try:
                    t_result = test_result[key]
                    break
                except KeyError:
                    print(f"key: {key} does not exists in list:", test_result.keys())
            if t_result is not None:
                smoothed_val = smooth(t_result, 0.99)
                plt.plot(smoothed_val)        
                plt.savefig(graph_path)
                plt.show()
```
